Developers/Newcomers_Guide.rst

```diff
@@ -125,14 +144,37 @@ ready to be accepted, you can optionally indicate this to other
 developers by starting your ``Pull Request`` title with ``wip``
 (case-insensitive).
 
-Step 7. Review Process
+Step 7. Waiting for Review
+--------------------------
+
+After creating a Pull Request, your PR is open to the review process (to read
+more about it, have patience and it is explained on the next step), and all
+you can do is wait. The best thing you can do while at this step is review
+other people's PRs. Not only will this help the maintainers with the work load,
```

Comment:
This should really be:
`This will not only help ...`
:D
\+ last `the` -> `their`
\+ `work load` -> `workload`